docs/workflows.md

```diff
@@ -47,23 +47,51 @@ The release workflow can be configured by tweaking the environment variables in
   <summary>Click here for a list of variables and how they're used.</summary>
 
   ```yaml
-  env:
-    # The base filename of the binary produced by `cargo build`.
-    BINARY: bevy_template
-    # The name to use for the packaged application produced by this workflow.
-    PACKAGE_NAME: bevy-template
-    # The itch.io page to upload to, in the format: `user-name/project-name`.
-    # Comment this out to disable.
-    ITCH_TARGET: the-bevy-flock/bevy-template
-    # The organization or author that owns the rights to the game.
-    OWNER: the-bevy-flock
-    # The path to the assets directory.
-    ASSETS_DIR: assets
-    # Whether packages produced by this workflow should be uploaded to the Github release.
-    UPLOAD_PACKAGES_TO_GITHUB_RELEASE: true
-    # Before enabling LFS, please take a look at GitHub's documentation for costs and quota limits:
-    # https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-storage-and-bandwidth-usage
-    USE_GIT_LFS: false
+  # The base filename of the binary produced by `cargo build`.
+  cargo_build_binary_name: bevy_quickstart
+
+  # The path to the assets directory.
+  assets_path: assets
+
+  # Whether to upload the packages produced by this workflow to a GitHub release.
+  upload_to_github: true
+
+  # The itch.io project to upload to in the format `user-name/project-name`.
+  # There will be no upload to itch.io if this is commented out.
+  upload_to_itch: the-bevy-flock/bevy-quickstart
```

Comment:
This is still using the old spelling further down this document

Reply:
Fixed, thank you